app/src/main/res/values/strings.xml

```diff
@@ -301,4 +301,6 @@
     <string name="app_shortcut_last_added_long">@string/last_added</string>
     <string name="app_shortcut_last_added_short">@string/last_added</string>
     <string name="playlist_is_empty">Playlist is empty</string>
+    <string name="snack_bar_action_undo">UNDO</string>
```

Comment:
No need for caps.

Reply:
That seems to be how google does it: https://developer.android.com/training/snackbar/action.html

Reply:
Hmm, I expected the snackbar to just transform it to uppercase by itself. That's sort of annoying.

Reply:
As far as I know the snackbar does transform it to uppercase. Otherwise, I would prefer String.toUppercase()

Reply:
Turns out it automatically converts it to uppercase. Fixed.